flipdisc/framework/app.py

```diff
@@ -13,6 +13,12 @@
 import zmq.eventloop.zmqstream
 
 import redis
```

Comment:
Just so our offline chat is recorded here, I think we're using the blocking API, but should be using the asyncio one:

```suggestion
import redis.asyncio as redis
```